server/mysterium_api.go

```diff
@@ -95,8 +96,6 @@ func (mApi *mysteriumAPI) FindProposals(nodeKey string) ([]dto_discovery.Service
 		return nil, err
 	}
 
-	log.Info(mysteriumAPILogPrefix, "FindProposals fetched: ", proposalsResponse.Proposals)
```

Comment:
All endpoints adds some logs after it's successful - removing this makes it inconsistent.
We should keep this log or remove all success logs in this class.

Reply:
Fixed